packages/packages.csv

```diff
@@ -173,6 +171,7 @@ Microsoft.Extensions.TaskCache.Sources,noship,exclude
 Microsoft.Extensions.TypeNameHelper.Sources,noship,exclude
 Microsoft.Extensions.WebEncoders,ship,include
 Microsoft.Extensions.WebEncoders.Sources,noship,exclude
+Microsoft.Extensions.WebSockets.Internal,noship,exclude
```

Comment:
Filed https://github.com/aspnet/SignalR/issues/24 to track a better Id for those packages and \ or an alternate solution. I'll leave them as noship for now